include/SFML/Graphics/Text.hpp

```diff
@@ -162,8 +162,13 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     /// \brief Set the letter spacing factor
     ///
     /// The default spacing between letters is defined by the font.
-    /// This method enables you to set a factor to the spacing
-    /// between letters. By default the letter spacing factor is 1.
+    /// This factor doesn't directly apply to the existing
+    /// spacing between each character, it rather adds a fixed
+    /// space between them which is calculated from the font
+    /// metrics and the character size.
+    /// Note that factors below 1 (including negative numbers) bring
```

Comment:
Well this is not entirely correct I would say. My proposal:
```cpp
/// This factor doesn't directly apply to the existing
/// spacing between each character. It rather uses a fixed
/// space which is calculated from the fonts whitespace 
/// and the character size. This spacing is scaled by the 
/// `spacingFactor` and added to the default spacing between 
/// two characters.
```

Reply:
> this is not entirely correct

Which part is incorrect?

> My proposal

Well, no need to describe all implementation details. Users don't need to know that.